packages/flutter/lib/src/gestures/events.dart

```diff
@@ -1657,3 +1659,42 @@ class PointerCancelEvent extends PointerEvent {
     );
   }
 }
+
+/// Determine the approriate hit slop pixels based on the [kind] of pointer.
+double computeHitSlop(PointerDeviceKind kind) {
+  switch (kind) {
+    case PointerDeviceKind.mouse:
+    case PointerDeviceKind.stylus:
```

Comment:
Unsure whether styles is more a mouse or touch device...

Reply:
yeah, I have no idea really. The only stylus I've ever used was the one that came with the Nintendo DS. In that case, it was more of a precision assist for the existing touch screen. I could imagine for a stylus like that it would not be disambiguated from a touch even.

Reply:
lol, I see you asked the same question above...

Reply:
And people often use stylii in place of mice when they use it on a drawing surface. I think of stylii as being much more precise than touch, so I'd go with the more precise slop on them.

Reply:
Good to know!